crypto/fipsmodule/bcm.c

```diff
@@ -228,7 +228,8 @@ static void BORINGSSL_bcm_power_on_self_test(void) __attribute__ ((constructor))
 #endif
 
 static void BORINGSSL_bcm_power_on_self_test(void) {
-#if !defined(OPENSSL_NO_ASM)
+#if !defined(OPENSSL_NO_ASM) && !defined(OPENSSL_PPC32BE) && !defined(OPENSSL_PPC64BE)
+// TODO: Do we need to implement this for PPC BE architectures?
```

Comment:
In my AES experiments, I left it as
```
#if !defined(OPENSSL_NO_ASM) && !defined(OPENSSL_PPC32BE)
```
Because testing on an emulated POWER8 (ISA 2.07) Big-endian 64-bit architecture that has VCRYPTO capability, it could use the assembly of [crypto/fipsmodule/aes/asm/aesp8-ppc.pl](https://github.com/aws/aws-lc/blob/9db98588b97078acdcda13d9f4b5b7be86e5b602/crypto/fipsmodule/aes/asm/aesp8-ppc.pl) and [crypto/fipsmodule/modes/asm/ghashp8-ppc.pl](https://github.com/aws/aws-lc/blob/9db98588b97078acdcda13d9f4b5b7be86e5b602/crypto/fipsmodule/modes/asm/ghashp8-ppc.pl)

Reply:
Per conversation with Nevine, I'll add a comment above this line:
```
// TODO: remove !defined(OPENSSL_PPC64BE) from the check below when starting to support
```